tests/build_/test_build_request.py

```diff
@@ -1267,14 +1267,34 @@ def test_render_prod_with_pulp_no_auth(self):
         with pytest.raises(OsbsValidationException):
             build_request.render()
 
+    def test_render_prod_with_empty_triggers(self, tmpdir):
+
+        self.create_image_change_trigger_json(str(tmpdir), custom_triggers=[])
+        build_request = BuildRequest(str(tmpdir))
+        kwargs = get_sample_prod_params()
+        build_request.set_params(**kwargs)
+        build_request.render()
+
     @staticmethod
-    def create_image_change_trigger_json(outdir):
+    def create_image_change_trigger_json(outdir, custom_triggers=None):
         """
         Create JSON templates with an image change trigger added.
 
         :param outdir: str, path to store modified templates
         """
 
+        triggers = custom_triggers if custom_triggers is not None else [
```

Comment:
I made a separate empty triggers test instead of parametrizing test_render_prod_request_with_trigger because that test only passes if a whole bunch of plugins load which are kicked off by the trigger. I thought it was less readable when the majority of the test is wrapped in an if/else on the parameters.